# Custom message file for publishing robot's position and velocity
float64 current_x
float64 current_y
float64 current_vx
float64 current_vy
